### DISCUSSION TITLE
Dashboard Copy Paste Duplicate Position

### DISCUSSION THREAD

**Discussion by milandzuris** [home-assistant/frontend#17191] [Dashboard]
Please add Copy Paste Duplicate and Position for Dashobards
![image](https://github.com/home-assistant/frontend/assets/42905545/b21c50fa-aa6b-4082-9a64-e837a910c976)
![image](https://github.com/home-assistant/frontend/assets/42905545/ffac0f2d-36e4-4360-a9ac-76c9a326796a)

**Comment by karwosts** [comment]
Sorry I don't understand what is being asked for, could you elaborate a bit?

You want to duplicate entire dashboards? What would you "paste" it into?

**Reply by milandzuris** [reply]
yes, copy to etc. phone dashboard, or dupilcate dashboard
